meta-resin-rocko/recipes-containers/tini/tini_%.bbappend

```diff
@@ -0,0 +1,2 @@
+# tini links with -static and because we don't have rcrt1.o in our glibc we disable PIE for it:
+SECURITY_CFLAGS_pn-${PN} = "${SECURITY_NOPIE_CFLAGS}"
```

Comment:
We already set  SECURITY_NO_PIE_CFLAGS which doesn't include SECURITY_PIE_CFLAGS which is defined based on GCCPIE. So given that we already set SECURITY_NO_PIE_CFLAGS in the recipe, the pie flags should not be set. Sounds like something else is happening here.

Reply:
Uh I see what's going on. The GCC defaults to pie now. But it sounds like a bug in yocto because the above mentioned variable, SECURITY_NO_PIE_CFLAGS, will still use pie due to the configure default to gcc.

Reply:
SECURITY_NO_PIE_CFLAGS has been deprecated in favor of SECURITY_NOPIE_CFLAGS in Poky Rocko

Reply:
Well it is still there but I see that it's not used anymore. And I see how it should work with SECURITY_NOPIE_CFLAGS.

So we should remove the entry in common layer and have per version bbappend from now on? I don't like the confusion with the common layer at this point. We either add SECURITY_NOPIE_CFLAGS in the common layer along with the old variable, or we make bbappens for each meta-resin yocto layer setting the correct value. Having this combination can bring confusion in the future.

Reply:
I will move it to the common layer instead